/usr/lib/python3/dist-packages/scipy/__init__.py:146: UserWarning: A NumPy version >=1.17.3 and <1.25.0 is required for this version of SciPy (detected version 1.26.3
  warnings.warn(f"A NumPy version >={np_minversion} and <{np_maxversion}"
[2024-11-14 21:03:41.029] [console] [info] TransmissionSim::registering Input 
HELICS Version: 3.5.1 (2024-03-19)
TransmissionSim: Federate TransmissionSim has been registered
TransmissionSim: Registered Publication ---> TransmissionSim/transmission_voltage
/usr/local/lib/python3.10/dist-packages/helics/capi.py:9090: UserWarning: This is deprecated. Use `helicsInputGetTarget` instead.
  warnings.warn("This is deprecated. Use `helicsInputGetTarget` instead.")
TransmissionSim: Registered Subscription ---> IEEE_123_feeder_0/totalLoad
[2024-11-14 21:03:41.680] [console] [debug] TransmissionSim (131072)[initializing]::Granting Initialization
[2024-11-14 21:03:41.680] [console] [debug] TransmissionSim (131072)[initializing]::Granted Time=-9223372036.854776
[2024-11-14 21:03:41.680] [console] [debug] TransmissionSim (131072)[executing]::Granting Execution
[2024-11-14 21:03:41.680] [console] [debug] TransmissionSim (131072)[0]::Granted Time=0
TransmissionSim: Substation Voltage to the Distribution System = 136.59 kV
[2024-11-14 21:03:41.763] [console] [debug] TransmissionSim[0]::setting value for TransmissionSim/transmission_voltage size 24
[2024-11-14 21:03:55.937] [console] [info] Transmission Federate (1879048192)[t=0]:: core tick
[2024-11-14 21:04:05.387] [console] [error] Transmission Federate (1879048192)[t=0]::lost comms with 1879048194
[2024-11-14 21:04:05.387] [console] [info] Transmission Federate (1879048192)[t=0]::sending disconnect
[2024-11-14 21:04:05.387] [console] [error] TransmissionSim (131072)[0]::lost comms with 1879048194
Traceback (most recent call last):
  File "/home/ubuntu/simple_gridlabd_example/Transmission/Transmission_simulator.py", line 151, in <module>
    grantedtime = h.helicsFederateRequestTime(fed, t)
  File "/usr/local/lib/python3.10/dist-packages/helics/capi.py", line 4717, in helicsFederateRequestTime
    raise HelicsException("[" + str(err.error_code) + "] " + ffi.string(err.message).decode())
helics.capi.HelicsException: [-101] lost comms with 1879048194
